libtracepoint-decode-cpp/include/tracepoint/PerfEventSessionInfo.h

```diff
@@ -55,7 +55,7 @@ namespace tracepoint_decode
         // to SetClockData, but the difference between them will be the same as the
         // difference between the values provided to SetClockData.
         void
-        GetClockOffset(
+        GetClockData(
```

Comment:
Note: The data returned by this method usually comes from the perf.data CLOCK_DATA header, so fix the name to align with the naming established by the perf.data file format spec.